scripts/workers/code_upload_submission_worker.py

```diff
@@ -518,6 +518,25 @@ def cleanup_submission(
         )
 
 
+def is_submission_container_executed(
```

Comment:
Please rename this method to `is_submission_evaluation_done`

Reply:
Updated